datalad/support/sshconnector.py

```diff
@@ -185,7 +187,7 @@ def socket_dir(self):
         if self._socket_dir is None:
             # TODO: centralize AppDirs (=> datalad.config?)
             from appdirs import AppDirs
-            self._socket_dir = AppDirs('datalad', 'datalad.org').user_config_dir
+            self._socket_dir = AppDirs('datalad', 'datalad.org').user_cache_dir
```

Comment:
It should be a subdirectory, and get `700` permsions. Subdir name could come from a config setting that has the above as a part. I think one like this is already defined.